ivy_tests/test_ivy/test_frontends/test_torch/test_blas_and_lapack_ops.py

```diff
@@ -675,6 +675,33 @@ def test_torch_pinverse(
         input=x[0],
         rcond=rtol,
     )
+    
+# trapz
+def test_torch_trapz(
+    dtype_y_x,
+    use_x,
+    dim,
+    dx,
+    on_device,
+    fn_tree,
+    frontend,
+    test_flags,
+):
+    dtype, y, x = dtype_y_x
+    if use_x:
+        test_flags.num_positional_args = 2
+        kwargs = {"y": y, "x": x, "dim": -1}
+    else:
+        test_flags.num_positional_args = 1
+        kwargs = {"y": y, "dx": dx, "dim": dim}
```

Comment:
I am not sure what you're trying to do here in general 🤔. The frontend helper `helpers.test_frontend_function` takes care of testing the function with a combination of args/kwargs and so there is no need to hardcode this into your test function itself. Any particular reason for adding this check?

Also, the frontend function `trapz` does not contain any argument called `dx` so why are you supplying it here?